js_modules/dagit/packages/core/src/ui/Page.tsx

```diff
@@ -1,7 +1,8 @@
 import styled from 'styled-components/macro';
 
 export const Page = styled.div`
+  height: 100%;
   overflow-y: auto;
-  padding: 16px 0 32px;
+  padding-bottom: 64px;
```

Comment:
Might be wrong but I think this needs a `box-sizing: border-box;` so that the height: 100% value is inclusive of the padding and this doesn't cause 64px of scrolling on pages?